backend/src/test/scala/com/softwaremill/adopttapir/starter/content/ContentServiceTest.scala

```diff
@@ -0,0 +1,23 @@
+package com.softwaremill.adopttapir.starter.content
+
+import com.softwaremill.adopttapir.starter.{Setup, StarterDetails}
+import com.softwaremill.adopttapir.template.ProjectGenerator
+import com.softwaremill.adopttapir.test.BaseTest
+
+class ContentServiceTest extends BaseTest {
```

Comment:
it would be good to have a test for one specific configuration checking something more than "it generated" - verifying some of the generated `Node` structure, content, etc. We aren't really testing anywhere that the generated `Node`s are correct

Reply:
additionally we need to have a separate test for the (quite complicated) merging logic, and write some really good test cases there

probably the whole filepath+content -> node conversion can be extracted to an `object` or sth like that and tested independently of the starter service etc.